osu.Game/Overlays/Toolbar/ToolbarRulesetSelector.cs

```diff
@@ -54,6 +59,9 @@ private void load()
                     }
                 }
             });
+
+            foreach (var ruleset in Rulesets.AvailableRulesets)
+                selectionSamples[ruleset.ShortName] = audio.Samples.Get($"UI/ruleset-select-{ruleset.ShortName}");
```

Comment:
You mentioned that custom rulesets can also use this, but they won't be able to with this logic as it stands. The resource would need to be read from their storage, not the game's one.

Not sure if this is an issue just yet though. Might be something we add support for in the future.

Reply:
Oh, I didn't realise that custom ruleset resources were independent... bad assumption on my behalf about how resources worked then, oops.

Reply:
Something similar to `CreateIcon()` is likely what we want, but I would probably recommend not doing that just yet due to concerns mentioned in #11240.